src/pages/graphql/payment-services-extension/mutations/create-vault-card-setup-token.md

```diff
@@ -7,7 +7,7 @@ description: This mutation creates a temporary token associated to the given pay
 
 <InlineAlert variant="info" slots="text" />
 
-This mutation is available only if you have installed [Payment Services for Adobe Commerce](https://commercemarketplace.adobe.com/magento-payment-services.html) 2.10.0 or higher.
+This mutation is available only if you have installed [Payment Services for Adobe Commerce](https://commercemarketplace.adobe.com/magento-payment-services.html) 2.10.0 or higher with the functionality [vault without a purchase](https://experienceleague.adobe.com/en/docs/commerce-merchant-services/payment-services/payments-checkout/vaulting#vaulting-without-purchase).
```

Comment:
Same. I think this confuses more than it clarifies. I propose to, instead, update the description and clarify that this mutation is to be used only when creating a vault card payment token. Example:

> title: createVaultCardSetupToken mutation
description: This mutation creates and returns a temporary setup token to be used as an input argument to the createVaultCardPaymentToken mutation when storing a payment method for a later purchase.